go.mod

```diff
@@ -392,5 +401,8 @@ require (
 	k8s.io/component-base v0.28.1 // indirect
 	k8s.io/kubectl v0.28.1 // indirect
 	oras.land/oras-go/v2 v2.2.1 // indirect
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.1.4 // indirect
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 )
+
+replace sigs.k8s.io/kubectl-validate => github.com/eddycharly/kubectl-validate v0.0.0-20230914071638-b53689140439
```

Comment:
Should we fork to kyverno org?

Reply:
I don't think we need for now as we don't really plan to maintain a fork.
The goal is to have changes merged upstream.

My fork only contains upstream `main` with changes from @fjogeleit PR (https://github.com/kubernetes-sigs/kubectl-validate/pull/65)